orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/pipeline/job/RunJobStage.java

```diff
@@ -76,6 +79,13 @@ public void taskGraph(@Nonnull StageExecution stage, @Nonnull TaskNode.Builder b
     }
   }
 
+  @Override
+  public void afterStages(@Nonnull StageExecution stage, @Nonnull StageGraphBuilder graph) {
+    if (stage.getContext().getOrDefault("noOutput", "false").toString().equals("true")) {
```

Comment:
Worth making this an equalsIgnoreCase (or a Boolean)?

Reply:
LGTM!